resources/generate_book.py

```diff
@@ -413,13 +413,19 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
             text('Squid can spawn in any deep ocean. They drop $(thing)Ink Sacs$(), and ink any player that gets too close. Some say that squids in deep, unexplored caves have strange properties.', 'Squid'),  # todo: squid renderer does not work without proper context...
             empty_last_page(),
         )),
-        entry('food_and_water', 'Food and Water', '', pages=(
-            text('Food and water...'),
-            empty_last_page(),
-            # todo: overview of how hunger and saturation works in tfc page
-            # todo: overview of water, thirst and drinking
-            # todo: overview of player nutrition and it's effects
-            # todo: introduction to different food sources, links to preservation
+        entry('food_and_water', 'Food and Water', 'tfc:food/orange', pages=(
+            text('In TerraFirmaCraft, not only must you manage your hunger, but you must manage your thirst. Hunger works similar to vanilla. Most pieces of food restore about a fifth of your hunger bar. Some pieces of food may restore a bit less, such as $(thing)Cattail Roots$(). Eating food also restores $(thing)Saturation$(), which can be thought of as how full you are. Some foods are very filling, so they keep you from losing hunger for longer, while some foods have a short-lived effect.'),
+            text('You will lose hunger just from regular gameplay. Hunger drains faster if you do things like sprint, swim, or become $(l:mechanics/size_and_weight#overburdening)Overburdened$(). Information about the $(thing)Saturation$(), $(thing)Water$(), as well as $(thing)Nutrients$() available from a food is available by hovering over it in your inventory. Using $(item)$(k:key.sneak)$() reveals the full tooltip.'),
+            image('tfc:textures/gui/book/tutorial/food_tooltip.png', text_contents='How to display a food information tooltip.'),
+            text('All foods have a tooltip with this information. The tooltip includes the $(l:mechanics/decay)Decay Date$() of the food, which may be extended with preservation. When viewing this information, it\'s important to realize that not all foods are good to eat! For example, $(thing)Dough$() is a food, but it has no Nutrition, Saturation, or Water value. Eating it would not do much good.'),
```

Comment:
I would clarify "good to eat". "Not good to eat" to me, sounds like rotten food. Maybe remove that specific description and just explain exactly, that dough is a food with no nutritional value, no saturation or hydration?